Description: Fix build failure with GCC 4.7
 - include/boost/polygon/polygon_90_set_data.hpp: Include
   "detail/polygon_sort_adaptor.hpp" for gtlsort.
Bug-Debian: http://bugs.debian.org/672727
Author: Matej Vela <vela@debian.org>
Last-Update: 2012-05-17

--- a/include/boost/polygon/polygon_90_set_data.hpp
+++ b/include/boost/polygon/polygon_90_set_data.hpp
@@ -16,6 +16,7 @@
 #include "detail/iterator_points_to_compact.hpp"
 #include "detail/iterator_compact_to_points.hpp"
 #include "polygon_traits.hpp"
+#include "detail/polygon_sort_adaptor.hpp"
 
 //manhattan boolean algorithms
 #include "detail/boolean_op.hpp"
